src/DynamoCoreWpf/Manipulation/DynamoManipulatorContext.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Dynamo.Controls;
+
+namespace Dynamo.Wpf.Manipulation
```

Comment:
Let's not put this in the WPF namespace unless we have to. I can very much see non-WPF apps wanting to create manipulators...